blockchain/chain/src/store/chain_store.rs

```diff
@@ -92,7 +92,7 @@ where
             let key = m.cid()?.key();
             let value = &m.marshal_cbor()?;
             if self.db.exists(&key)? {
-                return Err(Error::KeyValueStore("Keys exist".to_string()));
+                return Ok(());
```

Comment:
Why did this change from an Err to Ok?

Reply:
Because it was a bug, re: conversation above